client/blocks/post-share/style.scss

```diff
@@ -291,6 +291,7 @@ $section-border: solid 1px transparentize( lighten( $gray, 20% ), .5 );
 }
 
 .post-share .post-share__footer-items {
```

Comment:
Sidenote: I hate this class name. It isn't your fault at all. In fact, I think it's the mine one :-/